doc/ingest.md

```diff
@@ -86,6 +88,32 @@ The network indexer nodes expect that metadata begins with a `uvarint` identifyi
   * the proposed `uvarint` protocol is `0x3D0000`.
   * the following bytes are not yet defined.
 
+#### ExtendedMetadata
+
+The `ExtendedMetadata` field allows for specification of provider families, in cases where a provider operates multiple PeerIDs, perhaps with different access methods between them, but over the same database of content.
+
+```
+type ExtendedMetadata struct {
+    Family [ExtendedProvider]
+    Override bool
+}
+
+type ExtendedProvider struct {
+    Provider String
+    Addresses [String]
+    Metadata optional Bytes
+    Signature Bytes
+}
+```
+
+* If Metadata is not specified for an `ExtendedProvider`, the metadata for an individual `Advertisement` will be used instead.
+* If an `ExtendedProvider` listing is written with no `ContextID`, those peers will be returned for all published advertisements for the advertisement Publisher.
+  * If `Override` is set on an `ExtendedProvider` entry on an advertisement with a `ContextID`, that indicates that any set chain-level set of providers should not be returned for that context ID.
+  * If `Override` is not set on an entry for an advertisement with a `ContextID`, it will extended any set chain-level `ExtndedProviders`.
+* The `Signature` for an `ExtendedProvider` is made using the key of that extended `Provider`.
+  * The full advertisement object is serialized, with all instances of `Signature` replaces with an empty array of bytes.
+  * This serialization is then hashed, and the hash is then signed.
```

Comment:
Would this break the verification for indexer nodes that don't support `Extended*` features (yet)?

Reply:
I think it does - do we think that's okay?

Reply:
It is certainly undesirable and would make the sentences like "indexers which understand the `ExtendedMetadata` extension should ignore the `Provider`" a bit redundant, since indexers that do not understand it will cease to ingest any further ads from the point at which the extensions are advertised and would have to upgrade.

Can we avoid this? It would be nice if nodes that have not upgraded continue to be able to ingest while nodes that have, get the extra features.

We could:
* keep the ad signature calculation the same.
* Then include the ad signature in the `ExtendedProvider` signature calculation.

That means we can keep old nodes working, and avoid attacks like stitching `ExtendedProvider` from one ad onto another ad.

Do you think that would work?

Reply:
see if you think the current setup is reasonable

Reply:
yeah, that was my intention. i'll add a sentence saying that more clearly